code/modules/cargo/exports/gear.dm

```diff
@@ -58,26 +58,26 @@
 	export_types = list(/obj/item/clothing/head/radiation)
 
 /datum/export/gear/radsuit
-	cost = 100
+	cost = 50
```

Comment:
refund value if you accidentally (or not) send all back

Reply:
Why are you reducing radsuits then? I'd suggest going with the original prices for refunds if anything. https://github.com/tgstation/tgstation/pull/37833/files#diff-9600b7e33d7555351713821d4e2dfafcL109

Reply:
read the op